src/Uno.Toolkit.UI/Behaviors/ControlExtensions.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Windows.System;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+#endif
+
+namespace Uno.Toolkit.UI
+{
+	public static class ControlExtensions
+	{
+		private static ILogger _logger = typeof(ControlExtensions).Log();
+
+		#region DependencyProperty: Command
+
+		public static DependencyProperty CommandProperty { get; } = DependencyProperty.RegisterAttached(
+			"Command",
+			typeof(ICommand),
+			typeof(ControlExtensions),
+			new PropertyMetadata(default(ICommand), OnCommandChanged));
+
+		public static ICommand GetCommand(DependencyObject obj) => (ICommand)obj.GetValue(CommandProperty);
+		public static void SetCommand(DependencyObject obj, ICommand value) => obj.SetValue(CommandProperty, value);
+
+		#endregion
+
+		private static void OnCommandChanged(DependencyObject sender, DependencyPropertyChangedEventArgs e)
+		{
+			if (InputExtensions.IsEnterCommandSupportedFor(sender))
+			{
+				// for input controls, this will be implemented in InputExtensions.
+				InputExtensions.OnEnterCommandChanged(sender, e);
+			}
```

Comment:
That sounds kind of weird for me ...

Reply:
We are supposed to have 2 distinct behaviors: 
1. InputExtensions.EnterCommand can perform 3 things (switch focus, execute command, and/or close soft-keyboard).
2. ControlExtensions.Command execute the command only

Then, FT wanted to "rename" InputExtensions.EnterCommand into ControlExtensions.Command, and so we have this.
ControlExtensions.Command has different 3 implementations: ListView, NavigationView and Input Controls (Text/PasswordBoxes). They are already split in this ControlExtensions, except the input controls one also interacts with other InputExtensions properties, so the implemenation is placed there.